api/storage/app/lighthouse-schema.graphql

```diff
@@ -142,6 +142,9 @@ type Mutation {
   createAssessmentStep(poolId: UUID!, assessmentStep: AssessmentStepInput): AssessmentStep
   updateAssessmentStep(id: UUID!, assessmentStep: AssessmentStepInput): AssessmentStep
   deleteAssessmentStep(id: UUID!): AssessmentStep
+  createAssessmentResult(assessmentStepId: UUID!, poolCandidateId: UUID!, poolSkillId: UUID, assessmentResult: AssessmentResultInput!): AssessmentResult
```

Comment:
This looks a lot like the way we have been doing it so far.  Tristan is leaning towards having a single input parameter for mutations and not trying to reuse a shared input type between the "create" and "update" mutations.

Reply:
Changed in https://github.com/GCTC-NTGC/gc-digital-talent/pull/8302/commits/ea41bfb99141d0b23eeb6b2acbd252d64c74f834